documentation/testing/test-plan.md

```diff
@@ -1,106 +1,132 @@
 # Test Plan
 
-
 ## History and Approvals
 
-| Version | Name         | Role          | Date       | Remarks                                   |
-|---------|--------------|---------------|------------|-------------------------------------------|
-| 1.1     | Faizan Ahmad | Test Engineer | 29/11/2023 | need to be reviewed by other team members |
+| Version | Name         | Role         | Date       | Remarks                              |
+|---------|--------------|--------------|------------|--------------------------------------|
+| 1.1     | Faizan Ahmad | Test Engineer | 29/11/2023 | Need to be reviewed by other team members |
+| 1.2     | Faizan Ahmad | Test Engineer | 27/02/2024 |                                      |
 
 ## Project Overview
+
 ### Purpose
-The purpose of the software is to seamlessly integrate the functionalities of View My Financial Insights (VMFI) and School Financial Benchmarking (SFB). The end goal is to provide users with a unified platform where they can compare school financial data, view school spending, and analyse expenditures in comparison with similar schools. Another objective is to build ETL data pipelines to import and operate on raw data files and deposit them into some azure storage solution.
+The purpose of the software is to seamlessly integrate the functionalities of View My Financial Insights (VMFI) and School Financial Benchmarking (SFB). The end goal is to provide users with a unified platform where they can compare school financial data, view school spending, and analyse expenditures in comparison with similar schools. Another objective is to build ETL data pipelines to import and operate on raw data files and deposit them into some Azure storage solution.
```

Comment:
`Azure storage solution ` this is implementation details.

> Another objective is to a build robust ETL data pipelines, to import, transform and store raw data files.

Reply:
removed implementation details.